CONTRIBUTING.md

```diff
@@ -1,6 +1,24 @@
 # C#/WinRT Contributor's Guide
 
-Below is our guidance for how to report issues, propose new features, and submit contributions via Pull Requests (PRs). Please refer to [building and running C#/WinRT](./README.md#Building-and-running-cwinrt) to get started on building the repo.
+Below is our guidance for how to build the repo, report issues, propose new features, and submit contributions via Pull Requests (PRs). 
+
+## Building the C#/WinRT repo
+
+C#/WinRT currently requires the following packages, or newer, to build:
+
+- [Visual Studio 16.8](https://visualstudio.microsoft.com/downloads/) 
+- [.NET 5.0 SDK](https://dotnet.microsoft.com/download/dotnet/5.0) 
+- [.NET Core 2.1 SDK](https://dotnet.microsoft.com/download/dotnet-core/2.1)
+- [nuget.exe 5.8.0-preview.3](https://www.nuget.org/downloads)
+- Microsoft.WinUI 3.0.0-preview1.200515.3

```

Comment:
[3.0.0-preview4.210210.4](https://github.com/microsoft/CsWinRT/blob/ad6adcb997b40cea649cf4156451316e714bc3ea/src/Directory.Build.props#L11) ?

Reply:
Looks like we got a couple of versions referenced, but this is the one in the directory.props.  We should use this for now.  Its going to change once we move to use reunion anyways.